src/server/api/routes/vessel-reports.router.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
+
+// router setup
+
+const express = require("express");
+const router = express.Router({
+  mergeParams: true
+});
+const { authorizeUser } = require("../lib/middleware/auth.middleware");
+
+const ROLES = require("../../constants/roles");
+
+// controllers
+const vesselReportController = require("../controllers/vessel-report.controller");
+
+// ENDPOINT: /api/vessel_reports/ :POST
```

Comment:
Again naming conventions, but here we actually changed the path, so now the comment it just outdated. We have the API documentation now, so you are free to remove these comments if you want? Either that or make sure that it's up to date :)

Reply:
renamed vesselReportsController from vesselReportController .
// ENDPOINT: /api/vessel-reports/ :POST